deployment/ansible/group_vars/development

```diff
@@ -12,6 +12,10 @@ services_ip: "{{ lookup('env', 'RF_SERVICES_IP') | default('33.33.40.80', true)
 
 postgresql_host: "{{ services_ip }}"
 
+s3_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
 sqs_queue: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
 sqs_dead_letter_queue: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-dead-letter"
-s3_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}"
+logs_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-logs"
+artifacts_bucket: "raster-foundry-{{ lookup('env', 'USER') | regex_replace('^(.*\\\\\\)?(.*)$', '\\\\2') }}-artifacts"
```

Comment:
The bucket being used for artifact publishing in https://github.com/azavea/raster-foundry-tiler/pull/13 is `raster-foundry-releases`, but the ultimate path is going to be something like:

```bash
$ aws --profile rf-stg s3 ls s3://raster-foundry-releases/0.1.0/ | grep jar
2015-10-20 17:07:51  125567845 rf-tiler-assembly-0.1.0.jar
```